plank/laravel-media-manager
# #37 - Fix selected items

closes #37 

## Fix : 
I removed the `selectedElem` reset on `getDirectory` actions to keep previous selected elements.

## How to test : 
- Go to a folder and select a picture.
- Go to another folder and select another media. 

You should see both on the "carousel" bar.
